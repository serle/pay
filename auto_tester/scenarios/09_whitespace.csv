type, client, tx, amount
deposit, 1, 1, 100.0
withdrawal, 1, 2, 25.5
deposit, 2, 3, 50.25
