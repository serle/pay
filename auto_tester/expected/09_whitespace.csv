client,available,held,total,locked
1,74.5000,0.0000,74.5000,false
2,50.2500,0.0000,50.2500,false
